index.html

```diff
@@ -0,0 +1,82 @@
+								alt=""
+						/></a>
+					</li>
+				</ul>
+			</div>
+			<div class="resume">
+				<h1>
+					Hey There. <br />
+					<span>I'm Guishny</span>
+				</h1>
+				<h4>I'm a software Developer</h4>
+				<p class="desc">
+					I can help you to build beautiful projects, modern and responsive
+					websites, you can see my work on this page, if you are interested do
+					not hesitate to contact me
+				</p>
+			</div>
+			<div class="image">
+				<img
+					src="https://wizzy-techno.netlify.app/techno_img/7hexa1dark.png"
+					alt="Headline image"
+				/>
+			</div>
```

Comment:
- [ ] Kindly make sure your image is a background image using CSS background property currently you are using `<img>` tag, also ensure your design matches the given design.